networking/init-cms-bundle/4.0/config/packages/networking_init_cms.yaml

```diff
@@ -0,0 +1,101 @@
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/fos_ck_editor.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/fos_user.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/mopa_bootstrap.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/oneup_flysystem.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/oneup_uploader.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_admin.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_block.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_core.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_doctrine_admin.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_formatter.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_media.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_notification.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/sonata_user.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/stof_doctrine_extensions.yaml" }
+    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/symfony_cmf_routing_extra.yaml" }
+# Default security configuration
+#    - { resource: "@NetworkingInitCmsBundle/Resources/config/cms/security.yaml" }
+
+# Set templating to twig. FOSUserBundle requires templating to be setup
+framework:
+    templating:      { engines: ['twig'] }
+    cache:
+        pools:
+            page.cache:
+                adapter: cache.app #Setup page cache using template from cache.app
+
+# Set up the serializer to read config file for App Page and User entities
+jms_serializer:
+    metadata:
+        directories:
+            NetworkingSonataMediaBundle:
+                namespace_prefix: "Sonata\\MediaBundle"
+                path: "@NetworkingInitCmsBundle/Resources/config/serializer"
+            app:
+                namespace_prefix: "App"
+                path: "%kernel.project_dir%/config/serializer"
+
+# Translation bundle configuration, swap environment variables if mange_locales_only option set to false
+lexik_translation:
+    fallback_locale:  '%env(LOCALE)%'      # (required) default locale to use
+    managed_locales: [ '%env(LOCALE)%'] # !IMPORTANT: Change to strings and not environment variables
+    resources_registration:
+        type:                 all  # all | files | database
+        managed_locales_only: false # set to true to only include managed_locales
+
+
+networking_init_cms:
+    # Set a cookie in the fronted end for language. If disabled, the URL will be prefixed with the locale
+    allow_locale_cookie: true
+
+    # Simple php based cache for caching the rendered HTML in the frontend
+    cache:
+        activate: true #default false
+        cache_time: 86400 #default value
```

Comment:
A recipe is not a place where you dump all default values. 

Please remove

Reply:
This is not a dump of default values, these are the initial values set, which the user can change. It is nice to have it visible, rather than only being in the configuration.php file.

Reply:
Sorry, When I said "default values" I meant "initial values".

Reply:
As mentioned, we could leave the initial values away (they are anyway in the configuration.php file), but it is important that the users see it. We could leave all the configuation setting away for our particular bundle, there are initial values set for almost every service and parameter in our code. But, without a view of what and where to customise it, you are making the user experience, in my humble opinion, pretty crappy.